install/kubernetes/helm/istio/requirements.yaml

```diff
@@ -55,8 +55,8 @@ dependencies:
     version: 1.1.0
     condition: certmanager.enabled
     repository: file://../subcharts/certmanager
-  - name: istio-cni
-    version: ">=0.0.1"
-    repository: https://raw.githubusercontent.com/istio/istio.io/master/static/charts
-    condition: istio_cni.enabled
+    #  - name: istio-cni
```

Comment:
So far I can't seem to get this to work - maybe a new version of helm is needed ?